### PR TITLE
chore: cleanup remaining `internals.future` code

### PR DESCRIPTION
**Pull request by iuioiua**

### CODE REVIEW

File: runtime/js/40_fs_events.js [hunk 1]
```diff
@@ -21,13 +20,6 @@ class FsWatcher {
   #promise;
 
   constructor(paths, options) {
-    if (internals.future) {
-      ObjectDefineProperty(this, "rid", {
-        __proto__: null,
-        enumerable: false,
-        value: undefined,
-      });
```
Comment (bartlomieju) [inline runtime/js/40_fs_events.js:29]: This needs to stay

File: runtime/js/99_main.js [hunk 2]
```diff
@@ -850,9 +845,6 @@ function bootstrapMainRuntime(runtimeOptions, warmup = false) {
         nodeDebug,
       });
     }
-    if (internals.future) {
-      delete globalThis.window;
```
Comment (bartlomieju) [inline runtime/js/99_main.js:854]: Remove `window` from `98_global_scope_window.js`; you probably want to leave the `Window` interface though

Comment (bartlomieju) [reply runtime/js/99_main.js:854]: CC @lucacasonato am I correct here?

File: runtime/js/40_fs_events.js (hunk 1 above)
Comment (iuioiua) [reply runtime/js/40_fs_events.js:29]: This is news to me. Why?

Comment (bartlomieju) [reply runtime/js/40_fs_events.js:29]: Nvm, i read it wrong. Yeah this one goes.

**Review by bartlomieju — APPROVED** [review]

**Pull request merged**